docs/source/developer/contributing.rst

```diff
@@ -76,8 +83,78 @@ bot <https://github.com/apps/lock>`__ will lock the issue. If additional
 discussion is desired, or if the pull request doesn't fully address the
 locked issue, please open a new issue referencing the locked issue.
 
-Tag Issues with Labels
-^^^^^^^^^^^^^^^^^^^^^^
+New issues are subject to triage. A developer will triage your new issue 
```

Comment:
Maybe something like "A developer with triage permissions will..." to make it more specific.